content/02.author-sections.md

```diff
@@ -234,8 +234,6 @@ More information on the tool and its inception is available in the project manus
 
 <!-- Casey Greene -->
 
-### Casey Greene
```

Comment:
Please add back this header.  I believe was due to the timing of the pull request and when #5 was merged.